src/libslic3r/GCode.cpp

```diff
@@ -1663,50 +1664,6 @@ namespace DoExport {
 	    }
 	}
 
-    //BBS: add plate id for thumbnail generate param
```

Comment:
Nice refactor

Reply:
Can't take credit for that. This is a direct copy from PrusaSlicer 😄